lms/templates/sudo/sudo.html

```diff
@@ -0,0 +1,21 @@
+{% extends "main_django.html" %}
+{% load i18n %}
+
+{% block body %}
+<section style="margin: 0 auto; width: 480px; padding: 50px;">
+    <div class="inner-wrapper">
+        <header>
+          <h2 style="text-align: center;">{% trans "Confirm Your Password to Access the Instructor Dashboard" %}</h2>
```

Comment:
Dan, did you simply try searching for this in the codebase? And also just
testing to see if it works?
http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/internationalization/i18n.html#coverage-testing
- I'm a big fan of testing this out to actually see if it works.

On Tue, Jul 7, 2015 at 2:11 PM, Daniel Friedman <notifications@github.com>
wrote:

> In lms/templates/sudo/sudo.html
> <https://github.com/edx/edx-platform/pull/7404#discussion_r34071271>:
>
> > @@ -0,0 +1,21 @@
> > +{% extends "main_django.html" %}
> > +{% load i18n %}
> > +
> > +{% block body %}
> > +<section style="margin: 0 auto; width: 480px; padding: 50px;">
> > +    <div class="inner-wrapper">
> > +        <header>
> > +          <h2 style="text-align: center;">{% trans "Confirm Your Password to Access the Instructor Dashboard" %}</h2>
>
> I didn't realize that we support standard django templates (usually we use
> mako). @sarina <https://github.com/sarina> - can you confirm whether trans
> actually works? I've never seen this before in the platform.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-platform/pull/7404/files#r34071271>.
>